src/main/java/Calculator/StringAddCal.java

```diff
@@ -0,0 +1,52 @@
+package Calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCal {
+
+    public static int AddNumbers(String text) {
+        if(text == null) return 0;
+        if(text.isBlank()) return 0;
+        return getSum(getIntegers(getValues(text)));
+    }
+
+    private static int[] getIntegers(String[] values) {
+        int[] numbers = new int[values.length];
+        for (int i=0; i<values.length; i++){
+            numbers[i] = Integer.parseInt(values[i]); // 숫자가 아닌 문자의 경우 NumberFormatException 발생
+            if(isNegative(numbers[i]) ==true) throw new RuntimeException("Negative Number Included");
+        }
+        return numbers;
+    }
+
+    private static boolean isNegative(int number) {
+        if(number < 0) return true;
+        else return false;
```

Comment:
```suggestion
        return number < 0;
```

간단히 이렇게 구현할 수 있을것 같아요 :)